routes/api.php

```diff
@@ -35,3 +45,28 @@
         Route::post('/', StoreUserController::class);
         Route::delete('/{user}', DeleteUserController::class);
     });
+
+/*
+|--------------------------------------------------------------------------
+| Employee Routes
+|--------------------------------------------------------------------------
+*/
+Route::prefix('employees')
+    ->group(static function () {
+        Route::get('/', ListEmployeeController::class);
+        Route::post('/', StoreEmployeeController::class);
+    });
+
+/*
+|--------------------------------------------------------------------------
+| Task Routes
+|--------------------------------------------------------------------------
+*/
+Route::prefix('tasks')
+->group(static function () {
+    Route::get('/', ListTaskController::class);
+    Route::get('/{task}', GetTaskController::class)->withTrashed();
+    Route::post('/', StoreTaskController::class);
+    Route::patch('/', UpdateTaskController::class);
```

Comment:
```suggestion
    Route::patch('/{task}', UpdateTaskController::class);
```

I think you're missing the {task} parameter, which is necessary to indicate the task you're updating.

Reply:
Completely agree